docker/build.sh

```diff
@@ -32,9 +33,8 @@ common_flags="-DskipTests -PskipPython -Dlicense.skip -Dspotless.skip -ntp"
 
 build_docker_image() {
   module=$1
-  clean="${2:-true}"
   clean_cmd=""
-  if [ "$clean" == "true" ]; then
```

Comment:
please revert, this was done on purpose, on some modules is better to clean before building

Reply:
reverted